MonoGame.Framework/Graphics/Vertices/VertexPositionNormalTexture.cs

```diff
@@ -23,10 +23,15 @@ VertexDeclaration IVertexType.VertexDeclaration
                 return VertexDeclaration;
             }
         }
+
         public override int GetHashCode()
         {
-            // TODO: FIc gethashcode
-            return 0;
+            unchecked {

```

Comment:
MonoGame code is following Microsoft C# standard so you shouldn't use inline braces, instead move them to a new line:
```c#
unchecked
{
```